src/mapOfPeople.js

```diff
@@ -41,6 +41,17 @@
  */
 function mapOfPeople(people) {
   // write code here
+  let seconds = 0;
+  const room = new Map();
+  for (const person of people) {
```

Comment:
better to use `loop` where we can get `index`, like `forEach`. in this case you can remove variable `seconds`.